packages/extension/src/background/index.ts

```diff
@@ -15,6 +27,95 @@ const cacheAmplitudeDeviceId = async ({
   }
 };
 
+const excludedCompanionOrigins = [
+  'https://twitter.com',
+  'https://www.google.com',
+  'chrome-extension://',
+];
+
+const isExcluded = (origin: string) =>
+  excludedCompanionOrigins.some((e) => origin.includes(e));
+
+const sendBootData = async (req, sender) => {
+  if (isExcluded(sender?.origin)) {
+    return;
+  }
+  const cacheData = JSON.parse(localStorage.getItem('boot:local'));
```

Comment:
I think we have a generic function that fetches the local boot data.

Reply:
Not sure we need the cached data, to be honest. Because the companion boot is a requirement before we can render anything. So when we need to render something we already have the most up-to-date boot data available.

Reply:
This part is only for anonymous user syncing if they modify the theme in the extension and move over to a companion enabled website.

Reply:
I see ok then...